### PR TITLE
Update readme for CA 0.5.2

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @MaciekPytel

### CODE REVIEW

File: cluster-autoscaler/README.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ there is a big chance that it won't work as expected.
 
 # Notable changes:
 
+CA Version 0.5.2:
+* Fix problems with pods using persisten volume claims in scale up.
```
Comment (MaciekPytel) [inline cluster-autoscaler/README.md]: 1. s/persisten/persistent
2. I would link the original issue here (https://github.com/kubernetes/contrib/issues/2507).
3. The description is very vague. What do you think about something along the lines of 'Fix a bug causing pods using persistent volume claims to be ignored in scale up consideration'?

File: cluster-autoscaler/README.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ there is a big chance that it won't work as expected.
...
+
+CA Version 0.5.1:
+* Fix problems with slow network route creations on cluster scale up.
```
Comment (MaciekPytel) [inline cluster-autoscaler/README.md]: That describes the root cause of the issue, not the impact to user.

File: cluster-autoscaler/README.md (hunk 1 above)
Comment (mwielgus) [inline cluster-autoscaler/README.md]: Done.

File: cluster-autoscaler/README.md (hunk 2 above)
Comment (mwielgus) [inline cluster-autoscaler/README.md]: Done, added link to the problem.

**Comment by MaciekPytel** [comment]
lgtm

**Pull request merged**